fe/estimator_abfe.py

```diff
@@ -24,17 +24,16 @@
 class SimulationResult:
     xs: np.array
     boxes: np.array
```

Comment:
Tangential: noticed that `estimator_abfe` defines a simulation result containing `boxes`, but `estimator`'s definition does not includes `boxes`... Will open a separate issue